README.md

```diff
@@ -673,6 +673,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 #### Email Clients
 
 * [Airmail](http://airmailapp.com) - Fast email client. For both Mac OS and iOS.
+* [CanaryMail](https://canarymail.io/) - Secure email app for Mac and iPhone with built-in PGP Support and AI assitance. ![Freeware][Freeware Icon]
```

Comment:
The official website introduces an ios application
@antonengelhardt

Reply:
@jaywcjlove I have linked that app.